packages/core/helper-plugin/lib/src/content-manager/utils/tests/formatContentTypeData.test.js

```diff
@@ -1,14 +1,21 @@
-import formatComponentData from '../formatComponentData';
+import formatContentTypeData from '../formatContentTypeData';
 import testData from './testData';
 
 const { contentType, components, modifiedData } = testData;
 
-describe('STRAPI_HELPER_PLUGIN | utils | formatComponentData', () => {
-  it('should add the __temp_key__ property to each repeatable component object', () => {
+describe('STRAPI_HELPER_PLUGIN | utils | formatContentTypeData', () => {
+  it('should add the __temp_key__ property to each repeatable component object && stringify json fields', () => {
```

Comment:
Can you create another case for the JSON field so we don't mix tests?
I like the idea to have on case for one purpose only. WDYT?